applications/fishing-map/src/features/datasets/datasets.mock.ts

```diff
@@ -82,6 +82,33 @@ const datasets: Dataset[] = [
       },
     ],
     endpoints: [
+      {
+        id: 'carriers-multiple-vessel',
+        description: 'Endpoint to retrieve multiple vessels information',
+        downloadable: true,
+        pathTemplate: '/v1/vessels',
+        params: [],
+        query: [
+          {
+            label: 'Datasets',
+            id: 'datasets',
+            type: 'string',
+            required: true,
+          },
+          {
+            label: 'Ids',
+            id: 'ids',
+            type: 'string',
+            required: true,
+          },
+          {
+            label: 'binary',
```

Comment:
I've just checked it out and seems that  it could be also requested as binary to save some network data but I couldn't found the pbf decoder, I'll ask API team and, thanks for the catch!